doc/Installing-Connectors.md

```diff
@@ -4,7 +4,7 @@ Installing Connectors (Twitter/GNU Social)
 * [Home](help)
 
 
-Friendica uses addons to provide connectivity to some networks, such as Twitter or App.net. 
+Friendica uses addons to provide connectivity to some networks, such as Twitter or App.net.
```

Comment:
Is app.net still alive? Maybe this should contain another network instead as example.

Reply:
I have no idea if App.net is still in use.

Reply:
Me neither. Maybe @annando knows.

Reply:
app.net is dead.

Reply:
*sad trombone*

Reply:
So this should be changed accordingly then.